src/main/java/org/cloudfoundry/promregator/auth/AuthenticatorController.java

```diff
@@ -40,7 +38,7 @@ private void determineGlobalAuthenticationEnricher() {
 	}
 	
 	private void loadMapFromConfiguration() {
-		for (TargetAuthenticatorConfiguration tac : this.promregatorConfiguration.getTargetAuthenticators()) {
+		for (AuthenticatorConfiguration tac : this.promregatorConfiguration.getTargetAuthenticators()) {
```

Comment:
Is there a certain reason why you disliked the prefix `Target` here?

Reply:
We had 2 classes before `AuthenticatorConfiguration` and its sub-class `TargetAuthenticatorConfiguration` which added an `id` field. I ended up unifying the 2, since the empty `id` field doesn't cause trouble.